hal/ia32/cpu.c

```diff
@@ -161,54 +175,60 @@ int hal_cpuReschedule(spinlock_t *spinlock, spinlock_ctx_t *scp)
 			spinlock->dmin = spinlock->e - spinlock->b;
 		}
 
-		__asm__ volatile(
-			"xorl %%eax, %%eax; "
-			"incl %%eax; "
-			"xchgl %0, %%eax; "
-		:: "m" (spinlock->lock)
+		/* clang-format off */
+		__asm__ volatile (
+			"xorl %%eax, %%eax\n\t"
+			"incl %%eax\n\t"
+			"xchgl %0, %%eax\n\t"
+		:
+		: "m" (spinlock->lock)
 		: "eax", "memory");
+		/* clang-format on */
 
 		err = *scp;
 	}
 	else {
 		err = cpu_getEFLAGS();
 	}
 
-	__asm__ volatile(
-		"pushl %%eax; "
-		"pushl %%cs; "
-		"leal 1f, %%eax; "
-		"pushl %%eax; "
-		"xorl %%eax, %%eax; "
-		"call interrupts_pushContext; "
-
-		"call _interrupts_multilockSet; "
-
-		"leal (%%esp), %%eax; "
-		"pushl $0; "
-		"pushl %%eax; "
-		"pushl $0; "
-		"movl %1, %%eax; "
-		"call *%%eax; "
-		"cli; "
-		"addl $12, %%esp; "
-
-		"call _interrupts_multilockClear; "
-
-		"jmp interrupts_popContext; "
+	/* clang-format off */
+	__asm__ volatile (
+		"pushl %%eax\n\t"
+		"pushl %%cs\n\t"
+		"leal 1f, %%eax\n\t"
+		"pushl %%eax\n\t"
+		"xorl %%eax, %%eax\n\t"
+		"call interrupts_pushContext\n\t"
+
+		"call _interrupts_multilockSet\n\t"
+
+		"leal (%%esp), %%eax\n\t"
+		"pushl $0\n\t"
+		"pushl %%eax\n\t"
+		"pushl $0\n\t"
+		"movl %1, %%eax\n\t"
+		"call *%%eax\n\t"
+		"cli\n\t"
+		"addl $12, %%esp\n\t"
+
+		"call _interrupts_multilockClear\n\t"
+
+		"jmp interrupts_popContext\n\t"
 		"1: "
```

Comment:
Introduces unnecessary empty space ` ` after `1:` label.
```suggestion
		"1:"
```